stable/firehose/templates/deployment.yaml

```diff
@@ -81,7 +81,7 @@ spec:
         - mountPath: /work-dir
           name: workdir
       {{- end }}
-      {{if or (index .Values.telegraf.enabled) (index .Values "init-firehose" "enabled") (gt (len .Values.volumes) 0)}}
+      {{if or (index .Values.telegraf.enabled) (index .Values "init-firehose" "enabled") (gt (len .Values.secretsAsVolume) 0)}}
```

Comment:
nitpick 😬  : this name kinda sounds like multiple secrets into same volume, which isn't the case. we can maybe use: either `secretsAsVolumes` or `secretAsVolume` ?